### DISCUSSION TITLE
Is it safe to reduce existing Probes' step size?

### DISCUSSION THREAD

**Discussion by AlexDicy** [oetiker/SmokePing#369] [Q&A] [answered]
Hi, I am currently using Smokeping with FPing only.
This is my `Probes` file:
```
*** Probes ***

+ FPing
binary = /usr/bin/fping
step = 120


+ FPing6
binary = /usr/bin/fping
step = 120
protocol = 6
```
And my Database file:
```
*** Database ***

step     = 300
pings    = 20

# consfn mrhb steps total

AVERAGE  0.5   1  1008
AVERAGE  0.5  12  4320
    MIN  0.5  12  4320
    MAX  0.5  12  4320
AVERAGE  0.5 144   720
    MAX  0.5 144   720
    MIN  0.5 144   720

```
By looking at the charts, it looks like they are correctly updating every 120 seconds or 2 minutes:
![image](https://github.com/oetiker/SmokePing/assets/11839341/82f419e2-6213-4a37-ab70-5a0e8bb09a39)

Can I change my Probes' step from 120 to 60 or 30 without losing any data? Is it going to update and work for existing targets?
My goal is to have finer info on shorter cases of packet loss. Thank you!

**Comment by svenvg93** [comment, marked as answer]
Hi @AlexDicy , 
If you change the steps or pings in the Database file you will need to remove the current rrd files. Otherwise a error will be thrown on the webgui

`Error: RRD parameter mismatch ('Different number of data sources: /data/InternetSites/Youtube.rrd has 23, create string has 28'). You must delete /data/InternetSites/Youtube.rrd or fix the configuration parameters.`
